shared/hybrid/config.xml

```diff
@@ -24,9 +24,20 @@
         version   = "1.0.0">
     <name>Hybrid Application</name>
 
+    <!-- To allow XHR requests with the Whitelist plugin -->
+    <allow-navigation href="https://localhost" />
+    <allow-navigation href="https://*.force.com" />
+    <allow-navigation href="https://*.salesforce.com" />
+    <allow-navigation href="https://*.visualforce.com" />
+    <allow-navigation href="https://*.documentforce.com" />
+    <allow-navigation href="https://*.salesforce-communities.com" />
```

Comment:
Same question as the other repos -> what's `salesforce-communities.com`? I've never heard of it - I don't think we own that domain. That sounds like a custom community domain that a customer might have created.